fe/fe-core/src/main/java/com/starrocks/connector/ConnectorPartitionTraits.java

```diff
@@ -388,6 +396,17 @@ public List<Column> getPartitionColumns() {
         }
     }
 
+    /**
+     * Check whether the base table's partition has changed or not.
+     * NOTE: If the base table is materialized view, partition is overwritten each time, so we need to compare
+     * version and modified time.
+     */
+    private static boolean isBaseTableChanged(Partition partition,
+                                              MaterializedView.BasePartitionInfo mvRefreshedPartitionInfo) {
+        return partition.getVisibleVersion() != mvRefreshedPartitionInfo.getVersion()
```

Comment:
when partition is overwritten, is it partition ID also changed? if so, can we define a concept "HybridVersion" that resembles  to hybrid clock, for an example, use (PartitionId<<32) ||version; then compare the HybridVersion.

Reply:
Good idea. But it's a great job, to be simple, we use `visibleVersionTime` here since mv's version map has kept this since.